pdns/packethandler.cc

```diff
@@ -1320,6 +1336,74 @@ void PacketHandler::completeANYRecords(DNSPacket& p, std::unique_ptr<DNSPacket>&
   }
 }
 
+bool PacketHandler::tryAuthSignal(DNSPacket& p, std::unique_ptr<DNSPacket>& r, DNSName &target) {
+  DLOG(g_log<<Logger::Warning<<"Let's try authenticated DNSSEC bootstrapping (RFC 9615) ..."<<endl);
+  if(!d_dk.isSignalingZone(d_sd.zonename)) {
```

Comment:
Please check for a _signal label, before you do actual work,  like you do in pdnsutil or add an off switch. Metadata lookups are not free.